Heat_method_3/doc/Heat_method_3/Concepts/HeatMethodTraits_3.h

```diff
@@ -49,6 +49,8 @@ class HeatMethodTraits_3
   ///  Functor with operator: `FT operator()(const Point_3& p, const Point_3& q) const` which computes the squared distance between `p` and `q`.
   typedef unspecified_type Compute_squared_distance_3;
 
+  ///  Functor with operator: `FT operator()(const Vector_3& v) const` which computes the squared length of `v`.
+  typedef unspecified_type Compute_squared_length_3;
 
```

Comment:
So the weights package requires a change in the heat method package?

Reply:
The weights package requires an addition of one operator available in the traits class because it is used to compute the new version of the weight. The thing is that all Kernels have this operator so it is very small requirement imho.

Reply:
@sloriot do you consider this as a breaking change that must be documented in the changelog.  It only considers  users who write their own traits class, but still.

Reply:
No, it's like a bugfix I would say as it was probably already somehow needed.